.sync.yml

```diff
@@ -1,10 +1,6 @@
 ---
-.pmtignore:
-  paths:
-    - .puppet-lint.rc
 spec/spec_helper.rb:
   mock_with: ':mocha'
-Gemfile:
-  optional:
-    ':test':
-      - gem: puppet-lint-param-docs
```

Comment:
I think this line meant we used to check the param docs. Does removal here mean we stop checking that?

Reply:
that's now coming via vox-pupuli-puppet-lint-plugins